test/utils/runners.go

```diff
@@ -127,6 +127,7 @@ type RCConfig struct {
 	ReadinessProbe    *v1.Probe
 	DNSPolicy         *v1.DNSPolicy
 	PriorityClassName string
+	PodAnnotations    map[string]string
```

Comment:
nit: this is similar to labels, so probably it would fit better after Labels field